stable/sonatype-nexus/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: sonatype-nexus
-version: 0.1.8
-appVersion: 3.5.1
+version: 0.1.9
```

Comment:
I'd say that as soon as you add a variable, you're adding a new feature, in which case you should bump the minor version.

Reply:
If you are referring to `image.repository` that variable was already in the chart in deployment.yaml (https://github.com/kubernetes/charts/blob/master/stable/sonatype-nexus/templates/deployment.yaml#L30) but absent from the README.md

Reply:
Ah hm, you're right. Not sure what I was looking at...